content/en/docs/concepts/observability-primer.md

```diff
@@ -2,7 +2,7 @@
 title: Observability Primer
 description: Core observability concepts.
 weight: 9
-cSpell:ignore: KHTML
+cSpell:ignore: KHTML webshop
```

Comment:
Hmm. KHTML isn't used in the page anymore AFAICT.
`webshop` shouldn't be necessary since it only appears in code. Can you double check @svrnm?

Reply:
fixed, `webshop` is necessary because it's not in a code block but inline code which seems to be checked